_posts/2020-07-08-event-driven-functions-with-openfaas-and-nats.md

```diff
@@ -0,0 +1,146 @@
+---
+title: "Event driven functions with OpenFaaS and NATS"
+description: Vivek outlines how you can make use of NATS to trigger your functions in OpenFaaS through the new connector-sdk component.
+date: 2020-07-08
+image: /images/kafka-connector/aluminium-building.jpg
+categories:
+  - NATS
+  - tutorial
+  - examples
+author_staff_member: vivek
+dark_background: true
+---
+
+In this blog post, I will show how you can invoke OpenFaaS function in response to messages sent on NATS topics in publish-subscribe model.
+OpenFaaS functions are accessible over HTTP endpoints via gateway service but OpenFaaS provides several other way to invoke OpenFaaS functions with help of [connector-sdk](https://github.com/openfaas-incubator/connector-sdk). 
+
```

Comment:
I would add a statement similar to 

> The connector sdk provides a resuable and tested interface and implementation that allows developers to quickly create a new event source. The code that is unique to OpenFaaS is standardized so that the develop can focus on the details of the receiving message from the event source.

Reply:
Thanks for the suggestion, I have added it.